tests/unit/lib/foxbox/foxbox_test.js

```diff
@@ -0,0 +1,84 @@
+import Foxbox from 'js/lib/foxbox/foxbox';
+
+describe('Foxbox >', function() {
+  let foxbox;
+  const singleBox = [{
+    public_ip: '1.1.1.1',
+    client: 'abc',
+    message: JSON.stringify({
+      local_origin: 'https://local.abc.box.knilxof.org:3000',
+      tunnel_origin: 'null',
+    }),
+    timestamp: Math.floor(Date.now() / 1000),
+  }];
+  const multiBox = singleBox.concat([
+    {
+      public_ip: '2.2.2.2',
+      client: 'def',
+      message: JSON.stringify({
+        local_origin: 'http://local.def.box.knilxof.org:3000',
+        tunnel_origin: 'null',
+      }),
+      timestamp: Math.floor(Date.now() / 1000),
+    },
+  ]);
+
+  function getMockNet(single = true) {
```

Comment:
We'll need a better mock and in its own file, but I wanted to keep this patch small for now.